tests/listobjects/matrix_ttus.go

```diff
@@ -99,4 +99,91 @@ var ttus = []matrixTest{
 			},
 		},
 	},
+	{
+		Name: "ttus_recursive_alg_combined_w2",
+		Tuples: []*openfgav1.TupleKey{
+			// Direct, should always return
+			{Object: "ttus:recursive_w2_1", Relation: "ttu_recursive_alg_combined_w2", User: "user:w2_anne"},
+
+			// Satisfies the ttu_recursive_alg_combined relation needed in ttu_parent links below iff condition valid
+			{Object: "ttus:recursive_w2_1", Relation: "user_rel2", User: "user:w2_anne", Condition: xCond},
```

Comment:
should this be a different object since `ttus:recursive_w2_1` is already directly assigned in line 106?